activate

```diff
@@ -27,7 +27,14 @@
 DIR="/var/lib/eos-phone-home"
 if [ ! -d "${DIR}" ] || [ ! -w "${DIR}" ]; then
     echo "${DIR} missing or not writable"
-    exit 1
+
+    if [ "$EUID" == "0" ]; then
+      echo "Making ${dir}"
+      mkdir -p /var/lib/eos-phone-home
```

Comment:
Makes it work locally with `sudo` on a dev machine and won't hurt real deployments